test/extended/setup_compat.sh

```diff
@@ -0,0 +1,354 @@
+#!/bin/bash
+#
+# Starts an OpenShift master for compatibility testing
+
+
+# Launches a server for OpenShift compatibility testing
+# PREVIOUS_CONTROLLER=1 launches with previous version controllers
+# PREVIOUS_API=1        launches with previous version API server
+
+# Sets up a current master server with previous controllers
+function os::test::extended::setup_compatibility {
+  # os::log::stacktrace::install
+
+  # build binaries
+  if [[ -z $(os::build::find-binary ginkgo) ]]; then
+    hack/build-go.sh vendor/github.com/onsi/ginkgo/ginkgo
+  fi
+  if [[ -z $(os::build::find-binary extended.test) ]]; then
+    hack/build-go.sh test/extended/extended.test
+  fi
+  if [[ -z $(os::build::find-binary openshift) ]]; then
+    hack/build-go.sh
+  fi
+  if [[ -z "${PREVIOUS_VERSION:-}" ]]; then
+    if [[ -z $(os::build::find-binary previousversion) ]]; then
+      hack/build-go.sh tools/previousversion
+    fi
+    openshift_binary=$(os::build::find-binary openshift)
+    previous_version=$(os::build::find-binary previousversion)
+    export PREVIOUS_VERSION="$(${previous_version} ${openshift_binary})"
```

Comment:
I think I'm going to remove this in favor of a simple static value. Thinking about it some more, I realized that trying to calculate a previous version is silly.